src/bridge/settings/openedx/version_matrix.py

```diff
@@ -120,22 +107,16 @@ def get_item(cls, key: OpenEdxDeploymentName) -> DeploymentEnvRelease:
                 application="gradebook",
                 application_type="MFE",
                 release="sumac",
-                branding_overrides=default_branding_overrides,
             ),
             OpenEdxApplicationVersion(
                 application="learner-dashboard",
                 application_type="MFE",
                 release="sumac",
-                branding_overrides={
-                    "@edx/brand@npm": "@mitodl/brand-mitol-residential@latest",
```

Comment:
We also need to keep @edx/brand@npm override.

Reply:
Thanks for catching this, I just added it back.